apisix/stream/xrpc/runner.lua

```diff
@@ -70,9 +74,49 @@ local function put_req_ctx(session, ctx)
 end
 
 
+local function filter_logger(ctx, logger)
+    if not logger or not logger.filter or #logger.filter == 0 then
+        return false
+    end
+
+    local expr, err = expr.new(logger.filter)
```

Comment:
Better avoid using the module name as the var name.
And let's introduce a cache to avoid creating new obj per operations in the next PR.

Reply:
update